### DISCUSSION TITLE
zombie tags

### DISCUSSION THREAD

**Discussion by AlanHK** [enzo1982/freac#449] [Help & Support]
I have been making audiobook M4B files from sets of MP3.

I've done quite a few  but today had a problem I hadn't seen before.
See https://www.mediafire.com/file/0jjfmhw1czzzngc/Sins.rar/file
This has 2 MP3 files from a set. I split a larger file into chapters using MP3directcut.
This preserves tags in the original file. 

Then as usual I used MP3tag to set all the tags, including track numbers and titles.
These MP3s should have titles "Chapter 1" and "Chapter 2" and be track numbers 2 and 3 .
That's what every audio properties app, Mediainfo, foobar2000,  etc., shows.
But loading them into fre:ac 1.1.4, it shows both are titled "Chapter 2" and both track 01. Also shows both are 1:10:29 (the length of the file they were cut from). Actually they are 22 and 15 minutes.
Other tags seem to be what I set.
I tried rewriting the tags several times, but no change. Eventually I used an app "ID3remover" which does what its name suggests. 
Then I reapplied the tags and in those files fre:ac reads these correctly. 

There must be something odd in how these files were originally created. I can see "Processed by AaxAudioConverter" in the files.

**Comment by Lee-Carre** [comment]
@AlanHK

Display-captures (‘screen-shots’) would help. I can think of several possibilities as to why you might be seeing this.
